spec/smoke_tests/jobseekers_can_view_homepage_spec.rb

```diff
@@ -13,7 +13,7 @@
 
   context "Jobseeker visits vacancy page" do
     let(:smoke_test_domain) do
-      paas_environment = !ENV.include?("PAAS_ENVIRONMENT") || ENV["PAAS_ENVIRONMENT"].empty? ? "production" : ENV["PAAS_ENVIRONMENT"]
+      paas_environment = ENV["PAAS_ENVIRONMENT"]
```

Comment:
If `PAAS_ENVIRONMENT` should always be set now, it might be worth changing `ENV["PAAS_ENVIRONMENT"]` to `ENV.fetch("PAAS_ENVIRONMENT")` - this will raise an exception if it's not there and it might be easier to figure out when something goes wrong.